recipes/gdc-client/meta.yaml

```diff
@@ -0,0 +1,28 @@
+{% set version = "1.3.0" %}
+
+package:
+  name: gdc-client
+  version: {{ version }}
+
+build:
+  number: 0
+
+# TODO compile from source to be able to support older systems.
+source:
+  fn: gdc-client-{{ version }}.zip
+  url: https://gdc.cancer.gov/system/files/authenticated%20user/0/gdc-client_v1.3.0_Ubuntu14.04_x64.zip  # [linux64]
+  url: https://gdc.cancer.gov/system/files/authenticated%20user/0/gdc-client_v1.3.0_OSX_x64.zip  # [osx]
+  sha256: 5e3c42d00c047d5035e48df91a622b2cd48c5465cb5940f13a80c4f075c6c624  # [linux64]
+  sha256: 5c6c24cd880df6413cc06caa3f253348037e34eceb023603555f198915b880de  # [osx]
+
+test:
+  commands:
+    - gdc-client --help
+
+about:
+  home: https://gdc.cancer.gov/access-data/gdc-data-transfer-tool
+  license: Apache v2
+  summary: GDC Data Transfer Tool
+
+extra:
+  notes: We use a precompiled version of GDC that needs at least Ubuntu 14.x in case of Linux.
```

Comment:
This will not work afaik. We test on an old CentOS-6 compliant image, so these binaries will most likely not work.

Reply:
Yes, I know, I was still a bit tired when I created the PR ;-). I am working now on compiling it from source... I need to push some stuff to conda-forge first.